gslab_scons/misc.py

```diff
@@ -11,39 +11,42 @@
 import _exception_classes
 from size_warning import issue_size_warnings
 
-def scons_debrief(target, env):
-    '''Execute functions after SCons has built all targets'''
+def scons_debrief(args):

```

Comment:
This introduce backward incompatibility, since the user has to modify SConstruct. @Shun-Yang I don't have a great grasp on this function and you were the last one to touch it, can you add a quick fix?

Reply:
^ Comment as part of #130 #133

Reply:
We can change `args['log']` back into the function argument as `log = 'state_of_repo.log'` but the reason we moved it is that this looks more compact and natural.

Reply:
We had a discussion in the lab just now. Our solution is to provide a user-advisory that you cannot pass `env` and `log` to the function any longer and have to pass `args`, at which point we point to the current SConstruct in `template`. So we're fine with both 1) backward incompatibility and 2) coupled gslab_python/template for this particular problem.